docs/api_output.rst

```diff
@@ -35,7 +35,7 @@ RGBLED
 ======
 
 .. autoclass:: RGBLED(red, green, blue, active_high=True)
-    :members: on, off, red, green, blue, value
+    :members: on, off, blink, red, green, blue, value
```

Comment:
Hmm, good point - it may be an example of duck typing, but that's not a bad thing. There probably ought to be a toggle. On color/value - color's just an alias for value but looking at the other classes we generally prefer including the "device specific" aliases over the generic names, so yeah I ought to change that too. I'll bung this stuff in a misc commit for a bit later - just ploughing through LEDBarGraph, then it's onto the big pins merge, then I'll do the misc stuff.